tests/screenshot_gui.py

```diff
@@ -47,11 +47,8 @@ def runTest(self):
             raise Exception("No reasonable http proxy found, please add one to the board config")
 
         print("Using proxy %s" % proxy)
-        #print("Using FirefoxDriver")
-        #driver = lib.common.firefox_webproxy_driver(proxy)
-        #driver.maximize_window()
-        print("Using ChromeDriver")
-        driver = lib.common.chrome_webproxy_driver(proxy)
+        driver = lib.common.firefox_webproxy_driver(proxy)
```

Comment:
I guess there should be one more commit selects chrome or ffox based on the new options?